data/schema.js

```diff
@@ -1,4 +1,4 @@
-var getbabelRelayPlugin = require('babel-relay-plugin');
-var schema = require('./schema.json');
+var getbabelRelayPlugin = require('babel-relay-plugin')
+var schema = require('./schema.json')
 
-module.exports = getbabelRelayPlugin(schema.data);
\ No newline at end of file
+module.exports = getbabelRelayPlugin(schema.data, { substituteVariables: true })
```

Comment:
Shall remove, was trying to get something wild working before I took the current approach (use two identical fragments, but set them as a different variables in the queries).